src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -144,7 +144,7 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 	private _handleVSCodeSequence(data: string): boolean {
 		const didHandle = this._doHandleVSCodeSequence(data);
 		if (!this._hasUpdatedTelemetry && didHandle) {
-			this._telemetryService?.publicLog2<{ classification: 'SystemMetaData'; purpose: 'FeatureInsight' }>('terminal/shellIntegrationActivationSucceeded');
+			this._telemetryService?.publicLog2<{}, { owner: 'meganrogge' }>('terminal/shellIntegrationActivationSucceeded');
```

Comment:
Looks like the classification and purpose fields got cut off here?

Reply:
fyi @meganrogge